src/main/java/com/intuit/apl/engine/Context.java

```diff
@@ -250,6 +251,41 @@ public boolean containsAllIgnoreCase(ArrayList<String> source, String target) {
     return true;
   }
 
+  public boolean verifyRequest(){
+    return verifyRequest(subject) && verifyRequest(resource) && verifyRequest(action) && verifyRequest(environment);
+  }
+
+  private boolean verifyRequest(Map<String, String> map){
+    if(CollectionUtils.isEmpty(map)){
+      return true;
+    }
+
+    Map<String, String> verifyMap = getPrefixMap(map, "verify_");
```

Comment:
```suggestion
        Map<String, String> verifyMap = getPrefixMap(map, VERIFY_PREFIX);
```
We can move the literal to String Constant.